text/metrics/0146-metrics-prototype-scenarios.md

```diff
@@ -0,0 +1,220 @@
+# Scenarios for Metrics API/SDK Prototyping
+
+With the stable release of the tracing specification, the OpenTelemetry
+community is willing to spend more energy on metrics API/SDK. The goal is to get
+the metrics API/SDK specification to
+[`Experimental`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#experimental)
+state by end of 5/2021, and make it
+[`Stable`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#stable)
+before end of 2021:
+
+* By end of 5/2021, we should have a good confidence that we can recommend
+  language client owners to work on metrics preview release. This means starting
+  from 7/1/2021 the specification should not have major surprise or big changes
```

Comment:
nit: s/surprise/surprises/


Also should we make any statement about avoiding breaking changes as much as possible, without guaranteeing there won't be any breaking change?

Reply:
I've fixed the s/surprise/surprises nit.

Regarding breaking changes, I think we only have the "no breaking change" guarantee after we announce spec `Stable`. Before stable it will be `Experimental` so we can make breaking changes, we should always keep a high bar though.